src/types/composer.ts

```diff
@@ -789,17 +805,23 @@ export class TransactionComposer {
   }
 
   /** Build an ATC and return transactions ready to be incorporated into a broader set of transactions this composer is composing */
-  private buildAtc(atc: algosdk.AtomicTransactionComposer): algosdk.TransactionWithSigner[] {
+  private buildAtc(atc: algosdk.AtomicTransactionComposer): TransactionWithSignerAndContext[] {
     const group = atc.buildGroup()
 
     const txnWithSigners = group.map((ts, idx) => {
       // Remove underlying group ID from the transaction since it will be re-grouped when this TransactionComposer is built
       ts.txn.group = undefined
-      // If this was a method call stash the ABIMethod for later
+      // If this was a method call return the ABIMethod for later
       if (atc['methodCalls'].get(idx)) {
-        this.txnMethodMap.set(ts.txn.txID(), atc['methodCalls'].get(idx))
+        return {
```

Comment:
Left to right this is differnet behaviour?

Reply:
Now that we have an abstraction that can store this info alongside the algosdk.Transaction, we no longer need to stash it in state within the composer.